python/nav/models/oui.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from nav.models.fields import VarcharField
+
+
+class OUI(models.Model):
+    """Defines an OUI and the name of the vendor the OUI belongs to"""
+
+    vendor = VarcharField()
+    oui = models.CharField(max_length=17)
```

Comment:
```suggestion
    oui = models.CharField(max_length=17, unique=True)
```

I know that we are already checking it in the SQL, but this makes it easier to know that this is the case without having to look into the pure SQL.

Reply:
good shout